### DISCUSSION TITLE
How to select a specific scroll bar like "f" opening a link?

### DISCUSSION THREAD

**Discussion by dacec354** [brookhong/Surfingkeys#1991] [Q&A] [answered]
## In a page with a lot of scroll bars, it's hard to keep switching between scrollbars.

**Comment by GrewHair** [comment, marked as answer]
It's <kbd>;fs</kbd>, if you're using default key bindings. You can remap it to anything you want, of course.

**Reply by dacec354** [reply]
thanks a lot
